Makefile

```diff
@@ -100,6 +100,14 @@ $(HOST_BUILD_DIR)/crc-embedder: $(SOURCES)
 .PHONY: cross ## Cross compiles all binaries
 cross: $(BUILD_DIR)/macos-amd64/crc $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/windows-amd64/crc.exe
 
+.PHONY: containerized ## Cross compile from container
+containerized: clean
+	${CONTAINER_RUNTIME} build -t crc-build .
```

Comment:
I should have double checked before mentioning `CONTAINER_RUNTIME`, but `RUNTIME` seems more common than `CONTAINER_RUNTIME` in OpenShift components, https://github.com/openshift/enhancements/pull/638 for one such case (though the only other one is `api`)

Reply:
We already used this variable for the docs build container. just re-using it here

Reply:
Also, I am not a huge fan of just calling this `${RUNTIME}` as it is too generic... I can't say `JRE` here, right? ;-)